x-pack/qa/third-party/active-directory/src/test/java/org/elasticsearch/xpack/security/authc/ldap/ActiveDirectoryGroupsResolverTests.java

```diff
@@ -117,6 +155,12 @@ private void assertValidSidQuery(Filter query, String[] expectedSids) {
         }
     }
 
+    private Map<String, Object> resolve(Collection<Attribute> attributes, LdapMetadataResolver resolver) throws Exception {
```

Comment:
This method looks very specific to the added test and its name is rather generic. I suggest we move the code to `testResolveTokenGroupsSID` , WDYT?

Reply:
Changed